src/main/webapp/i18n/en/quizExercise.json

```diff
@@ -5,7 +5,7 @@
                 "title": "Quiz Exercises",
                 "createLabel": "Create Quiz",
                 "editLabel": "Edit Quiz",
-                "importLabel": "Import a Quiz",
+                "importLabel": "Import Quiz",
                 "exportLabel": "Export Quiz Exercises",
                 "createModelDragAndDrop": "Create Drag and Drop Model Quiz",
                 "createOrEditLabel": "Create or edit a Quiz Exercise"
```

Comment:
Where are those kind of labels used? programmingExercise has the `createOrEditLabel` as well. Would it make sense to remove the `a ` here as well?

Reply:
I couldn't find any usage for them. This kind of label is used for user management but not for any of the exercises.
Then they should probably be deleted